app/components/identity-steps/step-four.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { toastNotificationTimeoutOptions } from '../../constants/toast-notification';
+
+export default class IdentityStepsStepFourComponent extends Component {
+  @service toast;
+  @tracked Chaincode = 'Generate Chaincode';
+  @tracked isChaincodeClicked = false;
+  @tracked hideChaincode = true;
+  @tracked isCopyClicked = false;
+  @tracked isChaincodePageButtonDisabled = true;
+
+  @action async handleGenerateChaincode(e) {
+    e.preventDefault();
+    this.Chaincode = 'hv2hz3xh1h';
```

Comment:
why chain code is hardcoded??

Reply:
yes because not integrated with backend
only worked in UI part